playground/mocks/data/idp/idx/authenticator-verification-email.json

```diff
@@ -1,316 +1,192 @@
 {
-  "stateHandle": "02WTSGqlHUPjoYvorz8T48txBIPe3VUisrQOY4g5N8",
-  "version": "1.0.0",
-  "expiresAt": "2020-05-05T16:58:36.000Z",
-  "intent": "LOGIN",
-  "remediation": {
-    "type": "array",
-    "value": [
+  "stateHandle":"02WTSGqlHUPjoYvorz8T48txBIPe3VUisrQOY4g5N8",
+  "version":"1.0.0",
+  "expiresAt":"2020-06-22T20:28:04.000Z",
+  "intent":"LOGIN",
+  "remediation":{
+    "type":"array",
+    "value":[
       {
-        "rel": [
+        "rel":[
           "create-form"
         ],
-        "name": "challenge-authenticator",
-        "href": "http://localhost:3000/idp/idx/challenge/answer",
-        "method": "POST",
-        "accepts": "application/vnd.okta.v1+json",
-        "value": [
+        "name":"challenge-authenticator",
+        "relatesTo":[
+          "$.currentAuthenticatorEnrollment"
+        ],
+        "href":"http://localhost:3000/idp/idx/challenge/answer",
+        "method":"POST",
+        "accepts":"application/vnd.okta.v1+json",
+        "value":[
           {
-            "name": "credentials",
-            "form": {
-              "value": [
+            "name":"credentials",
+            "type":"object",
+            "form":{
+              "value":[
                 {
-                  "name": "passcode",
-                  "label": "One-time verification code",
-                  "secret": true
+                  "name":"passcode",
+                  "label":"Enter code"
                 }
               ]
-            }
+            },
+            "required":true
           },
           {
-            "name": "stateHandle",
-            "required": true,
-            "value": "02WTSGqlHUPjoYvorz8T48txBIPe3VUisrQOY4g5N8",
-            "visible": false,
-            "mutable": false
+            "name":"stateHandle",
+            "required":true,
+            "value":"02WTSGqlHUPjoYvorz8T48txBIPe3VUisrQOY4g5N8",
+            "visible":false,
+            "mutable":false
           }
         ],
         "relatesTo": ["$.currentAuthenticatorEnrollment"]
       },
       {
-        "rel": [
+        "rel":[
           "create-form"
         ],
-        "name": "select-authenticator-authenticate",
-        "href": "http://localhost:3000/idp/idx/challenge",
-        "method": "POST",
-        "accepts": "application/vnd.okta.v1+json",
-        "value": [
+        "name":"select-authenticator-authenticate",
+        "href":"http://localhost:3000/idp/idx/challenge",
+        "method":"POST",
+        "accepts":"application/vnd.okta.v1+json",
+        "value":[
           {
-            "name": "authenticator",
-            "type": "object",
-            "options": [
-              {
-                "label": "Okta Password",
```

Comment:
super nit:
I was trying to use same authenticator list from `authenticator-verification-select-authenticator.json` for all `authenticator-verification-{{AUTHENTICATOR_NAME}}.json`.
Avoid change if unnecessary.

Reply:
I prefer it this way since I got the JSON from my postman which is a realistic representation of email verification remediation. Also, the one from `authenticator-verification-select-authenticator.json` has a lot more options and `relatesTo` objects within that which are absent here in my mock. I will have to change those as well, not sure what value it may add other than consistency in JSON mocks.

Reply:
> has a lot more options and relatesTo objects

This is intentional. `options` are mocked but `relatesTo` will be added (Sam is working on it)

Reply:
I will revisit once Sam's work is complete. It will be an easier change then...